packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,126 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 5px);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-small-horizontal-inset, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    height: 100%;
+    overflow: auto;
+    padding: var(--notification-dialog-body-padding, 1.25rem);
+    width: 100%;
+  }
+
+  .notification-dialog-body-text {
+    background-color: var(--notification-dialog-body-color, #fff);
+    font-size: var(--notification-dialog-body-font-size, 1.25rem);
```

Comment:
Also looks like the font-size here in unneeded since it is being set again in children both for `.title` (line 92) and `.message` (line 103) and could be removed.